bahmni-commons-omod/src/test/java/org/bahmni/module/bahmnicommons/web/v1_0/controller/search/BahmniAppointmentsPatientSearchControllerTest.java

```diff
@@ -0,0 +1,76 @@
+package org.bahmni.module.bahmnicommons.web.v1_0.controller.search;
+
+import org.bahmni.module.bahmnicommons.BaseIntegrationTest;
+import org.bahmni.module.bahmnicommons.contract.patient.response.PatientResponse;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.Matchers;
+import org.mockito.Mock;
+import org.openmrs.*;
+import org.openmrs.api.PatientService;
+import org.openmrs.module.webservices.rest.web.resource.impl.AlreadyPaged;
+import org.openmrs.module.webservices.rest.web.response.ResponseException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+public class BahmniAppointmentsPatientSearchControllerTest extends BaseIntegrationTest {
+    
+    @Mock
+    private HttpServletRequest httpServletRequest;
+
+    @Mock
+    private HttpServletResponse httpServletResponse;
+
+    @Mock
+    private PatientService patientService;
+    
+    @Autowired
+    private BahmniAppointmentsPatientSearchController bahmniAppointmentsPatientSearchController;
+
+    @Rule
+    public ExpectedException exceptionRule = ExpectedException.none();
+    
+    @Before
+    public void setUp() throws Exception {
+        bahmniAppointmentsPatientSearchController.setPatientService(patientService);
+        List<Patient> patients = new ArrayList<>();
+        Patient patient = new Patient();
+        patient.setPersonId(1);
+        patients.add(patient);
+        when(patientService.getPatients(Matchers.anyString())).thenReturn(patients);
+    }
+    
+    @Test
+    public void shouldSearchByIdentifier() {
```

Comment:
The OpenMRS convention would be to prefix this with the tested method name: `search_shouldSearchByIdentifier`. Same remark for the other tests.

Reply:
@mks-d, updated to follow naming convention as mentioned.